code/procs/jobprocs.dm

```diff
@@ -686,6 +686,15 @@
 						qdel(src.limbs.r_arm.remove(0))
 				boutput(src, "<b>Your singular arm makes you feel responsible for crimes you couldn't possibly have committed.</b>" )
 
+	if (src.traitHolder && src.traitHolder.hasTrait("nolegs"))
+		if (src.limbs)
+			SPAWN(6 SECONDS)
+				if (src.limbs.l_leg)
+					src.limbs.l_leg.delete()
+				if (src.limbs.r_leg)
+					src.limbs.r_leg.delete()
+			src.put_in_hand_or_drop(new /obj/stool/chair/comfy/wheelchair)
```

Comment:
I'd probably just set it to the mobs turf by doing:
`new /obj/stool/chair/comfy/wheelchair(get_turf(src))`

Reply:
Done,